README.md

```diff
@@ -37,7 +37,9 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 
     `.\configure.ps1`
 
-4. Build with
+4. Close the Powershell window and open a new one. 
```

Comment:
Can you please explain why?
Create a matching issue to discuss before we accept this change.